integration/pull_test.go

```diff
@@ -108,7 +108,7 @@ func TestOptimizeConsistentSociArtifact(t *testing.T) {
 			containerImage: "nginx:latest",
 		},
 		{
-			name:           "soci artifact is consistently built for drupal",
+			name:           "soci artifact is consistently built for alpine",
 			containerImage: "alpine:latest",
```

Comment:
Do we want to change `name` to alpine or change the `containerImage` to drupal?

Reply:
If we're going to change the actual container being tested, I think that would deserve its own PR and discussion. This version of the change has no practical impact, which is why I felt comfortable including it here.